src/main/java/at/ac/tuwien/student/sese2017/xp/hotelmanagement/config/SecurityConfig.java

```diff
@@ -0,0 +1,39 @@
+package at.ac.tuwien.student.sese2017.xp.hotelmanagement.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@EnableWebSecurity
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+  @Override
+  protected void configure(HttpSecurity http) throws Exception {

```

Comment:
Explain functionality in detail and javadocs

Reply:
done